toonz/sources/toonz/mainwindow.h

```diff
@@ -205,22 +205,23 @@ class MainWindow final : public QMainWindow {
   QAction *createViewerAction(const char *id, const char *name,
                               const QString &defaultShortcut,
                               const char *iconSVGName = "",
-                              QString newStatusTip = "");
+                              QString newStatusTip    = "");
   // For command bar, no shortcut keys
   QAction *createVisualizationButtonAction(const char *id, const char *name,
                                            const char *iconSVGName = "",
-                                           QString newStatusTip = "");
+                                           QString newStatusTip    = "");
 
   QAction *createMiscAction(const char *id, const char *name,
                             const char *defaultShortcut,
                             QString newStatusTip = "");
   QAction *createToolOptionsAction(const char *id, const char *name,
                                    const QString &defaultShortcut,
-                                   QString newStatusTip = "");
+                                   QString newStatusTip = "",
+                                   const char *iconSVGName = "");
```

Comment:
Note to self: I will submit a followup PR to reverse the order of these 2 parameters and update the impacted calls so the order matches all the other create action definitions.